libs/editor/WordPressEditor/src/main/res/values/strings.xml

```diff
@@ -154,6 +154,13 @@
     <string name="dialog_body_try_block_editor" translatable="true">We’ll be removing the classic editor for new posts soon, but this won’t affect editing any of your existing posts or pages. Get a head start by enabling the Block Editor now in site settings.</string>
     <string name="dialog_dismiss_try_block_editor" translatable="true">Dismiss</string>
 
-
+    <!-- Featured image replacement -->
+    <string name="replace_current_title">Replace current?</string>
```

Comment:
Wdyt about tweaking the title to not include an implied reference to the body (`current`)? I feel it might be tougher to translate. Some ideas: "Confirm replace?" or just "Replace" even?

While at it, maybe all the string names could be prefixed with `featured_image_` to provide context?

Reply:
I agree that, upon reflection, the names here a bit confusing! I've made some changes so that they're now prefixed with `featured_image_replace_dialog_`, following the lead of some of the other strings for dialogs in the `strings.xml` file. Do you think these new names seem okay?

Reply:
> Do you think these new names seem okay?

Yeap, names look good to me!

Just making sure, do you have thoughts about rephrasing the string as well?

Reply:
Ah, yes, I agree with the reasoning for rephrasing. I went with `Replace?` in https://github.com/wordpress-mobile/WordPress-Android/pull/14503/commits/b4dc8252d0706ec223a38614546cffbb94d5c292, as that seemed to flow best in my mind, let me know if you think anything else is needed here.

Reply:
Yeah, "Replace?" works for me too, thanks!